jatin/postgres_assignment/school.sql

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
please use appropriate datetype for each of the fields and create index if required on columns

Reply:
okay sir, i will try to name each datatype accordingly